src/Discord.Net.Core/Entities/Messages/IUserMessage.cs

```diff
@@ -57,6 +57,18 @@ public interface IUserMessage : IMessage
         /// </returns>
         Task UnpinAsync(RequestOptions options = null);
 
+        /// <summary>
+        ///     Publish (crosspost) this message.
+        /// </summary>
+        /// <param name="options">The options to be used when sending the request.</param>
+        /// <returns>
+        ///     A task that represents the asynchronous operation for publishing this message.
+        /// </returns>
+        /// <remarks>
+        ///     Publishing (crossposting) is only available in news channels.
```

Comment:
Remarks should describe what the method does, not just what to look out for. A better way to phrase this is 

```xml
<remarks>
<note type="warning">
This method will throw an [insert exception xref here] if attempted to publish in a non-news channel.
</note>
This method will crosspost the message. Please note that publishing or crossposting is only available in news channels.
</remarks>
```

Reply:
Sorry for wasting your time on this! Appreciate the guidance. Modified verbiage.